src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-// write your code here
+const body = document.body;
+const logo = document.querySelector('.logo');
+
+const promise1 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('Promise was rejected!'));
+  }, 3000);
+});
+
+function success(message) {
+  body.insertAdjacentHTML('beforeend', `
+    <div class="message">
+      ${message}
+    </div>
+  `);
+}
+
+function error(message) {
```

Comment:
success and error functions are alike. I recommend to create one function showMessage(message, class) {}